test/.config/nvim/init.vim

```diff
@@ -1 +0,0 @@
-../../vimrc
```

Comment:
You shouldn't need to change this. Undo these changes. This is just for Travis CI, which can follow symlinks.